rob1:
  coordinated_motion_controller:
    type: coordinated_motion_controllers/CoordinatedRobotController
    joints:
      - rob1_joint1
      - rob1_joint2
      - rob1_joint3
      - rob1_joint4
      - rob1_joint5
      - rob1_joint6

    positioner_link: positioner
    base_link: rob1_base_link
    eef_link: rob1_tool0
  

rob2:
  coordinated_motion_controller:
    type: coordinated_motion_controllers/CoordinatedRobotController
    joints:
      - rob2_joint1
      - rob2_joint2
      - rob2_joint3
      - rob2_joint4
      - rob2_joint5
      - rob2_joint6

    positioner_link: positioner
    base_link: rob2_base_link
    eef_link: rob2_tool0


positioner:
  positioner_controller:
    type: coordinated_motion_controllers/PositionerController
    joints:
      - positioner_joint

    coordinated_controllers:
      - /rob1/coordinated_motion_controller
      - /rob2/coordinated_motion_controller
